apps/fabric-website/index.html

```diff
@@ -80,39 +105,35 @@
     })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
     ga('create', 'UA-84911004-1', 'auto');
     ga('send', 'pageview');
-</script>
-<script type="text/JavaScript">
-        var varSegmentation = 0;
-        var varClickTracking = 1;
-        var varCustomerTracking = 1;
-        var varAutoFirePV = 1;
-        var Route = "50000";
-        var Ctrl = "SD300";
-        document.write("<script type='text/javascript' src='" + (window.location.protocol) + "//c.microsoft.com/ms.js'" + "'><\/script>");
-    </script>
-<noscript><img alt="" width="1" height="1" src="http://c.microsoft.com/trans_pixel.aspx" /></noscript>
-<script>
+  </script>
+  <script type="text/javascript">
+    var varSegmentation = 0; var varClickTracking = 1; var varCustomerTracking = 1; var varAutoFirePV = 1; var Route = "50000";
+    var Ctrl = "SD300"; var script = document.createElement('script'); script.setAttribute('type', 'text/javascript'); script.src
+      = (window.location.protocol) + "//c.microsoft.com/ms.js"; document.getElementsByTagName('head')[0].appendChild(script);
+  </script>
+  <noscript><img alt="" width="1" height="1" src="http://c.microsoft.com/trans_pixel.aspx" /></noscript>
+  <script>
     // OfficeDev Google Analytics
     (function (i, s, o, g, r, a, m) {
-      i['GoogleAnalyticsObject'] = r; i[r] = i[r] || function () {
+      i['GoogleAnalyticsObject '] = r; i[r] = i[r] || function () {
         (i[r].q = i[r].q || []).push(arguments)
       }, i[r].l = 1 * new Date(); a = s.createElement(o),
         m = s.getElementsByTagName(o)[0]; a.async = 1; a.src = g; m.parentNode.insertBefore(a, m)
-    })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
+    })(window, document, 'script ', '//www.google-analytics.com/analytics.js ', 'ga ');
 
-    if (window.location.host.indexOf('localhost') != -1 || window.location.host.indexOf('azurewebsites') != -1) {
-      ga('create', 'UA-65962112-1', 'auto'); //staging and Dev GA
+    if (window.location.host.indexOf('localhost ') != -1 || window.location.host.indexOf('azurewebsites ') != -1) {
```

Comment:
why add all these spaces or is it only the diff? 

![image](https://cloud.githubusercontent.com/assets/2201819/23976834/bc85ec78-09c0-11e7-9f88-dff4d0e73110.png)